src/mlpack/core/optimizers/sgd/sgd.hpp

```diff
@@ -105,7 +107,7 @@ class SGD
    * @param resetPolicy Flag that determines whether update policy parameters
    *                    are reset before every Optimize call.
    */
-  SGD(const double stepSize = 0.01,
+  SGD(const double stepSize = 0.001,
```

Comment:
Can you revert the step size change here? At the end it everyone should tune the step size anyway, but that's what we used before, so I think we should stick with ``0.01`` for now.

Reply:
Done. Thanks for the catch :)